src/MainWindow.vala

```diff
@@ -132,7 +137,12 @@ public class Camera.MainWindow : Gtk.ApplicationWindow {
 
         camera_content.set_player (camera_view);
 
-        loading_view.set_status (_("Connecting to \"%s\"…").printf (camera_view.get_camera_device ().get_name ()));
+         if  (camera_view.get_camera_device ().get_name () == null) {
```

Comment:
I wonder if we need this condition here because in line 115 we substitute `camera_devices[0].get_name ().get_name ()` whose consequence should be equivalent to `camera_view.get_camera_device ()`. I guess we can emit the condition and simplify:

```diff
diff --git a/src/MainWindow.vala b/src/MainWindow.vala
index 72869bf..507fde7 100644
--- a/src/MainWindow.vala
+++ b/src/MainWindow.vala
@@ -137,12 +137,7 @@ public class Camera.MainWindow : Gtk.ApplicationWindow {
 
         camera_content.set_player (camera_view);
 
-         if  (camera_view.get_camera_device ().get_name () == null) {
-            loading_view.set_status (_("Connecting to %s…").printf ("camera"));
-        } else {
-            loading_view.set_status (_("Connecting to \"%s\"…").printf (camera_view.get_camera_device ().get_name ()));
-        }
-    
+        loading_view.set_status (_("Connecting to %s…").printf (camera_name));
     }
 
     private void on_fullscreen () {
```